### ISSUE TITLE
Manual grading version of notebook produced by autograde is missing grade entry boxes and marker comment field

### ISSUE THREAD

**Issue by AstroMike** [jupyter/nbgrader#1546]
nbgrader version 0.6.2

jupyterhub --version
1.4.2

jupyter notebook --version
6.4.3

### Expected behavior

After autograding, the student's answer is wrapped in blue box with "Student's answers" and a place to enter the marks, then the student's code and a place for the marker to enter comments.

### Actual behavior

After autograding, the cell just shows the student's answer: there is no place to enter marks or comments

### Steps to reproduce the behavior

Hard to reproduce. Only 1 or 2 student's assignments show this behaviour, rest of the student submissions are fine. No python errors in the students assignments ... nor when I rerun autograde from the command line.

**Comment by perllaghu** [comment]
If it's just a couple of students out of a whole class, I suspect they have added cells.... and nbgrader doesn't know about them.

The Notebook Document files (the `.ipynb` files) are actually just JSON documents - so you can examine them with any editor that doesn't try to be clever & "render" them.

Compare your `source` notebook to the students `submitted` notebook... and check the JSON for the affected cells & compare the two files.

I suspect you'll find that your `source` file will include some metadate a bit like:
```
  'metadata': {'deletable': False,
                'nbgrader': {'cell_type': 'code',
                            'checksum': '98f0602b716b1846fcd998f6f67580c0',
                            'grade': False,
                            'grade_id': 'cell-5dc080c7963fae78',
                            'locked': False,
                            'schema_version': 3,
                            'solution': True,
                            'task': False}},
``` 
.... whilst the student's answer is in a cell without it.

(Having said that, I would also have expected other errors to show.... however it would be good to confirm the relevant cells _have_ metadata)

**Comment by AstroMike** [comment]
That's a great tip @perllaghu ... I looked at the students notebooks (there were 2 on this week's assignment) and it looks like all of the metadata had been stripped out of _all_ cells! 

But its a mystery how this happened ... one student says he exclusively worked in Jupyter Hub and didn't manually delete any of this. The other developed on his home machine then cut and paste into the assignment on Jupyter Hub.

So I am baffled as to how this happened.

I am confused as to why no errors are thrown in this case, either at the validate stage or by autograde.

**Comment by jhamrick** [comment]
> I am confused as to why no errors are thrown in this case, either at the validate stage or by autograde.

Yeah it's a bit unclear to me as well why this happened for them, but it's probably related to what @perllaghu suggested---perhaps they accidentally deleted cells and recreated them without realizing what happened, or something along those lines.

Agree it would be nice though if nbgrader could print a warning or something that this has happened at least to the instructor to say "it looks like some cells are missing so this might not be formatted as you expect".
